vhdbuilder/packer/install-dependencies.sh

```diff
@@ -241,20 +209,8 @@ fi
 ls -ltr /opt/gpu/* >> ${VHD_LOGS_FILEPATH}
 
 installBpftrace
-echo "  - bpftrace" >> ${VHD_LOGS_FILEPATH}
```

Comment:
this also doesn't come from os packages, might be easier to run `bpftrace -version` or something